src/components/modules/epics/ArticleCountOptOut.tsx

```diff
@@ -0,0 +1,215 @@
+import React, { useState, useRef, useEffect } from 'react';
+import { css } from 'emotion';
+import { brand } from '@guardian/src-foundations/palette';
+import { textSans } from '@guardian/src-foundations/typography';
+import { space } from '@guardian/src-foundations';
+import { Button } from '@guardian/src-button';
+import { SvgClose } from '@guardian/src-icons';
+import { ThemeProvider } from 'emotion-theming';
+import { brand as brandTheme } from '@guardian/src-foundations/themes';
+import { from } from '@guardian/src-foundations/mq';
+import { addCookie } from '../../../lib/cookies';
+
+const ARTICLE_COUNT_OPT_OUT_COOKIE = {
+    name: 'gu_article_count_opt_out',
+    daysToLive: 90,
+};
+
+const DAILY_ARTICLE_COUNT_STORAGE_KEY = 'gu.history.dailyArticleCount';
+const WEEKLY_ARTICLE_COUNT_STORAGE_KEY = 'gu.history.weeklyArticleCount';
+
+const optOutContainer = css`
+    display: inline-block;
+
+    ${from.tablet} {
+        position: relative;
+    }
+`;
+
+const articleCountButton = css`
+    background: none;
+    border: none;
+    padding: 0;
+    cursor: pointer;
+    text-decoration: underline;
+    font-family: inherit;
+    font-size: inherit;
+    font-weight: inherit;
+`;
+
+const overlayContainer = css`
+    position: absolute;
+    z-index: 100;
+    left: ${space[4]}px;
+    right: ${space[4]}px;
+    color: #ffffff;
+    background: ${brand[400]};
+    ${textSans.medium()}
+    padding: ${space[3]}px;
+
+
+    ${from.tablet} {
+        width: 325px;
+        left: 0;
+        padding-bottom: ${space[4]}px;
+    }
+`;
+
+const overlayHeader = css`
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+`;
+
+const overlayHeaderText = css`
+    font-size: 17px;
+    font-weight: bold;
+`;
+
+const overlayBody = css`
+    margin-top: ${space[1]}px;
+    font-size: 15px;
+`;
+
+const overlayCtaContainer = css`
+    margin-top: ${space[5]}px;
+    > * + * {
+        margin-top: ${space[3]}px;
+    }
+
+    ${from.mobileMedium} {
+        display: flex;
+
+        > * + * {
+            margin-top: 0;
+            margin-left: ${space[3]}px;
+        }
+    }
+`;
+
+const overlayNote = css`
+    margin-top: ${space[3]}px;
+    ${textSans.xsmall()}
+    font-style: italic;
+    display: none;
+
+    ${from.tablet} {
+        display: block;
+    }
+
+    a {
+        color: #ffffff !important;
```

Comment:
are these `important` because of DCR overrides?

Reply:
This was actually being overriden by something in contributions service. I'm trying to track down where the override comes from now. Without that the colour is set to #C70000 (error.400, red)

Reply:
Really struggling to find where this comes from